_data/events_gdcr2018/Almaty.json

```
@@ -0,0 +1,18 @@
+{
+  "title": "GDCR18 in Almaty",
+  "url": "https://events.epam.com/events/global-day-of-coderetreat-1",
+  "moderators": [
+    "@ssshogunnn",
+    "@EPAMKazakhstan"
```

I think that maybe EPAMKazakhstan is more an sponsor than a moderator.
```suggestion
  ],
  "sponsors": [
    "@EPAMKazakhstan"
  ],
```